src/Renderer/ThreeExtended/GLTFLoader.js

```diff
@@ -12,7 +12,7 @@ import * as THREE from 'three';
  * @author Takahiro / https://github.com/takahirox
  */
 
-THREE.GLTFLoader = ( function () {
+GLTFLoader = ( function () {
```

Comment:
You need to export it. `export default (function() {...}` would be enough I guess